From 5e7d67c23ad5090cc8119ebd177bb87d51e23cd1 Mon Sep 17 00:00:00 2001
From: Charlie Paul <cpaul.windriver@gmail.com>
Date: Tue, 1 Sep 2015 21:19:08 -0700
Subject: [PATCH 45/59] drivers/ethernet/lsi: Fixed code to support 4.1

git://git.yoctoproject.org/linux-yocto-4.1 standard/axxia/base
	commit 3dc58ee11164b3c1cc6c497786a060cd6d7d789e

Changed code to support the Linux 4.1

Signed-off-by: Charlie Paul <cpaul.windriver@gmail.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/ethernet/lsi/Makefile      |    4 ++--
 drivers/net/ethernet/lsi/lsi_acp_net.c |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/lsi/Makefile b/drivers/net/ethernet/lsi/Makefile
index 5ebc246..c981d39 100644
--- a/drivers/net/ethernet/lsi/Makefile
+++ b/drivers/net/ethernet/lsi/Makefile
@@ -1,6 +1,6 @@
 # Makefile for the LSI FEMAC network interface.
 
-obj-$(CONFIG_NET_LSI_MDIO) += lsi-mdio.o
-obj-$(CONFIG_NET_LSI_FEMAC) += lsi-femac.o
+#obj-$(CONFIG_NET_LSI_MDIO) += lsi-mdio.o
+#obj-$(CONFIG_NET_LSI_FEMAC) += lsi-femac.o
 obj-$(CONFIG_LSI_NET) += lsi_acp_mdio.o
 obj-$(CONFIG_LSI_NET) += lsi_acp_net.o
diff --git a/drivers/net/ethernet/lsi/lsi_acp_net.c b/drivers/net/ethernet/lsi/lsi_acp_net.c
index 5a284a5..9bc5cc3 100644
--- a/drivers/net/ethernet/lsi/lsi_acp_net.c
+++ b/drivers/net/ethernet/lsi/lsi_acp_net.c
@@ -954,7 +954,7 @@ static int appnic_open(struct net_device *dev)
 	napi_enable(&pdata->napi);
 
 	/* Install the interrupt handlers. */
-	return_code = request_irq(dev->irq, appnic_isr, IRQF_DISABLED, LSI_DRV_NAME, dev);
+	return_code = request_irq(dev->irq, appnic_isr, 0x00, LSI_DRV_NAME, dev);
 	if (0 != return_code) {
 		pr_err("%s: request_irq() failed, returned 0x%x/%d\n",
 		       LSI_DRV_NAME, return_code, return_code);
-- 
1.7.5.4

